drawee/src/main/java/com/facebook/drawee/generic/GenericDraweeHierarchy.java

```diff
@@ -585,14 +557,10 @@ public void setBackgroundImage(@Nullable Drawable drawable) {
    * @param drawable background image
    */
   public void setOverlayImage(int index, @Nullable Drawable drawable) {
-    // Note: we do not have mOverlaysIndex to reduce the memory footprint of this class.
```

Comment:
This two lines of comments are still relevant: we do rely on overlays being the last ones.